mrp_production_request/models/stock_rule.py

```diff
@@ -79,6 +79,14 @@ def _run_production_request(
             )
 
         # create the MR as SUPERUSER because the current user may not
+        # to avoid everytime MR creation
+        existing_MRs = request_obj.search(
```

Comment:
I also do not agree with this, if you are thinking on orderpoints this is already handled in _quantity_in_progress. Maybe I'm missing some detail here? but it seems to me quite custom process if you need this check.

Reply:
Hi Lois ... yes you are right , was my mistake and I  checked your PR and agree with your solution.